cegs_portal/templates/base.html

```diff
@@ -1,82 +1,155 @@
+                                </svg>
+                            </button>
+                        </div>
+                        {% if request.user.is_authenticated %}
+                        <div class="flex">
+                            <a
+                                href="{% url 'users:detail' request.user.username  %}"
+                                class="block text-md px-2 py-2 rounded text-white ml-2 font-bold hover:text-white mt-4 hover:bg-blue-700 lg:mt-0 no-underline"
+                                style="color: white;"
+                                ><i class="bi bi-person-circle"></i> {% translate "My Profile" %}</a
+                            >
 
-    </div>
+                            <a
+                                href="{% url 'account_logout' %}"
+                                class="block text-md px-2 ml-2 py-2 rounded text-white font-bold hover:text-white mt-4 hover:bg-blue-700 lg:mt-0 no-underline"
+                                style="color: white;"
+                            >
+                                <i class="bi bi-door-open"></i> {% translate "Sign Out" %}</a
+                            >
+                        </div>
+                        {% else %}
+                        <div class="flex">
+                            <a
+                                href="{% url 'account_signup' %}"
+                                class="block text-md px-2 py-2 rounded text-white ml-2 font-bold hover:text-white mt-4 hover:bg-blue-700 lg:mt-0 no-underline"
+                                style="color: white;"
+                                ><i class="bi bi-people-fill"></i> {% translate "Sign Up" %}</a
+                            >
 
-    <div class="flex flex-row flex-auto">
-      <div class="min-w-[5em] grow"></div>
-      <div class="flex flex-col">
-        {% block content %}
-          <p>Use this document as a way to quick start any new project.</p>
-        {% endblock content %}
-      </div>
-      <div class="min-w-[5em] grow"></div>
-    </div>
+                            <a
+                                href="{% url 'account_login' %}"
+                                class="block text-md px-2 py-2 rounded text-white font-bold hover:text-white mt-4 hover:bg-blue-700 lg:mt-0 no-underline"
+                                style="color: white;"
+                            >
+                                <i class="bi bi-person-badge"></i> {% translate "Login" %}</a
+                            >
+                        </div>
+                        {% endif %}
+                    </div>
+                </nav>
+            </div>
 
-    <footer class="mt-24 bg-dark p-4 text-white flex-none">© 2021 Duke University</footer>
-    </div>
-    {% block modal %}{% endblock modal %}
+            <div class="flex flex-row flex-auto">
+                <div class="min-w-[5em] grow"></div>
+                <div class="flex flex-col" style="margin-top: 80px">
```

Comment:
I think you should remove the `style` attribute here

Reply:
Will tailwind css class be fine as a replacement? The margin is needed because fixed navbars have a tendency to cut off the top of the page. Like this attached image shows.
![image](https://user-images.githubusercontent.com/19657615/224350051-ca712c4f-f98c-464a-b42e-a475fa894cf4.png)

Reply:
I updated it before I created the branch but not since yesterday. Will pull on this branch.